src/azure_keyvault_browser/__init__.py

```diff
@@ -1,7 +1,14 @@
-import importlib.metadata
 import sys
 
-if (sys.version_info[0], sys.version_info[1]) < (3, 9):
-    sys.exit("keyvault-browser requires Python 3.9 or later.")
+if (sys.version_info[0], sys.version_info[1]) < (3, 7):
+    sys.exit("azurecr-browser requires Python 3.7 or later.")
```

Comment:
Should this be `azure-keyvault-browser`?

Reply:
Clearly not had enough coffee today. Fixed in 3810a0cdd35f029572f0f93fce9e829de855272d.